src/compare/index.ts

```diff
@@ -46,7 +49,12 @@ export class Comparator {
   async *compare(pact: Pact): AsyncGenerator<Result> {
     if (!this.#router) {
       await parseOas(this.#oas);
-      this.#router = setupRouter(this.#oas);
+      for (const [key, value] of Object.entries(this.#oas.info)) {
+        if (key.startsWith("x-opc-config-")) {
+          this.#config.set(key.substring(13), value);
+        }
+      }
```

Comment:
@tuan-pham I got distracted and didn't get around to committing this.

See https://swagger.io/docs/specification/v3_0/openapi-extensions/